launcher/nodehandler.go

```diff
@@ -311,10 +311,12 @@ func (node *nodeHandler) getRequestedCPU(
 	instanceIdent aostypes.InstanceIdent, serviceConfig aostypes.ServiceConfig,
 ) uint64 {
 	requestedCPU := uint64(0)
+	cpuQuota := serviceConfig.Quotas.CPUDMIPSLimit
```

Comment:
ditto for the intermediate variable and clamping

Reply:
Somehow my comments about clamping for all cases disappeared: clamping should be done for all cases.

Reply:
fixed